components/Common/Footer.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
+/* eslint react/jsx-filename-extension: 0 */
+/* eslint react/jsx-no-literals: 0 */
+/* eslint arrow-body-style: 0 */
+// eslint-disable-next-line react/jsx-no-comment-textnodes
+// eslint-disable-next-line react/jsx-no-undef
+// eslint-disable-next-line jsx-a11y/anchor-is-valid
+
+import React from 'react';
+import Head from 'next/head';
+import './common.scss';
+import { HOME_STRINGS } from '../constants';
+
+const {
+    NUMBER,
+} = HOME_STRINGS;
+
+const Footer = () => {
+    return (
+        <React.Fragment>
+            <Head>
+                <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossOrigin="anonymous" />
+            </Head>
+            <div className="footer">
+                <div className="footer1 app-content" id="contact">
+                    <img className="logo logo-footer" src="/static/img/Group 2.png" alt="Smiley face" />
+                    <div className="footer1-text">
+                        <p className="footer1-text1">
+                            Lorem ipsum dolor sit amet, conse ctetur adipiscing
+                            <br />
+                            {' '}
+                            elit. Curabitur ante leo, finibus quis est ut, tempor
+                            <br />
+                            {' '}
+                            tincidunt ipsum.
+                        </p>
+                        <span className="reachout-container">
+                            <i className="fas fa-phone-volume phone" />
+                            {' '}
+                            { NUMBER }
+                        </span>
+                        <br />
+                        <span className="reachout-container">
+                            <i className="far fa-envelope-open envelope" />
+                              office@lifeaid.com
+                        </span>
+                        <br />
+                        <span className="reachout-container">
+                            <i className="fas fa-map-marker-alt address" />
+                              8 Kelvin Str. Los Angeles, CA
+                        </span>
+                    </div>
+                    <div className="footer1-text2 footer-texts">
+                        <p className="footer1-text2-header">Useful Links</p>
+                        <div className="useful-links-wrapper">
+                            <ul className="useful-links">
+                                <li>
+                                    <a>Testimonials</a>
+                                </li>
+                                <li><a>FAQ</a></li>
+                                <li><a>Terms & Conditions</a></li>
+                                <li><a>Our Partners</a></li>
+                                <li><a>Services</a></li>
```

Comment:
save these strings in an array and map over them to create this section of the component
for example:
```
const LIST_OF_STRINGS = ['hello', 'world']
...
<ul>
    LIST_OF_STRINGS.map(string => (
        <li><a>{string}</a></li>
    ))
</ul>
```